src/test/ui/dyn-keyword/dyn-2015-edition-keyword-ident-lint.rs

```diff
@@ -3,7 +3,8 @@
 // to detect or fix uses of `dyn` under a macro. Since we are testing
 // this file via `rustfix`, we want the rustfix output to be
 // compilable; so the macros here carefully use `dyn` "correctly."
-
+//
+// edition:2015
```

Comment:
We don't need it as the default edition is 2015.

Reply:
Oops, I really meant to explain that better and then forgot. We don't need it, but when the test is looking at edition specific behavior, it's a lot clearer to have a marker in the test that shows what edition it runs on. I'm pretty sure I saw them on some of the other tests that addressed 2015 edition specific behavior. Plus, we might change the default edition for these tests in the future. If new code is going to be written on a newer edition, it would make sense for most tests to look at the behavior of the current edition rather than the old one. Given all that, I thought it made sense to add it to the tests I was changing anyway, and then I put it on the others in the same group for consistency.